Makefile

```diff
@@ -5,7 +5,7 @@
 CABAL_BIN := cabal
 
 THRIFT1 := thrift1
-CABAL := $(CABAL_BIN) $(CABAL_CONFIG_FLAGS) $(GETDEPS_CABAL_FLAGS)
+CABAL := $(CABAL_BIN) $(CABAL_CONFIG_FLAGS) $(GETDEPS_CABAL_FLAGS) -f-bundled-folly -f-fbthrift
```

Comment:
Don't set flags here. They're supposed to be set in CABAL_CONFIG_FLAGS, which is set by the CI workflow, or cabal.project.

Reply:
Also, fbthrift should be enabled for the getdeps build

Reply:
> Don't set flags here. They're supposed to be set in CABAL_CONFIG_FLAGS, which is set by the CI workflow, or cabal.project.

Cannot easily set them in the internal CI workflow, already tried that. Re cabal.project, isn't it shared with the GitHub CI workflow?

Reply:
Last I checked the breakage was in fbthrift itself, CI was passing on the PR when it was landed.

Reply:
> Last I checked the breakage was in fbthrift itself, CI was passing on the PR when it was landed.

true, but it is fixed now

Reply:
The current error is cabal complaining about a leading zero in the folly-clib version number, I can put up a fix for that. https://github.com/facebookincubator/hsthrift/actions/runs/16402006410/job/46342785108

This PR won't work because the cabal CI job will fail - it needs `bundled-folly` to be enabled: https://github.com/facebookincubator/hsthrift/actions/runs/16374421038/job/46270656525?pr=160

Maybe it's possible for me to run the getdeps build locally, I haven't tried that yet.

Reply:
@pepeiborra what's the command to run the getdeps build? I put it on the internal wiki somewhere but I don't remember what it was exactly. There were some extra flags IIRC.

Reply:
getdeps.py build hsthrift --allow-system-packages --shared-libs --extra-cmake-def
ines='{"BOOST_LINK_STATIC": "OFF"}'